toonz/sources/toonzlib/tlog.cpp

```diff
@@ -19,8 +19,10 @@
 #else
 #include <syslog.h>
 #include <unistd.h>
+#ifndef FREEBSD
 #include <sys/timeb.h>
 #endif
```

Comment:
I'm pretty sure you can remove this include, it only provides `ftime` and that isn't used anywhere in this file.